packages/cli/src/abstract-server.ts

```diff
@@ -85,8 +83,6 @@ export abstract class AbstractServer {
 		this.endpointWebhookTest = this.globalConfig.endpoints.webhookTest;
 		this.endpointWebhookWaiting = this.globalConfig.endpoints.webhookWaiting;
 
-		this.uniqueInstanceId = generateHostInstanceId(instanceType);
```

Comment:
We were creating two unique IDs for the same process, once at `BaseCommand.setInstanceQueueModeId` and once in `AbstractServer.constructor`.